### PR TITLE
Responsive banner

### PR DESCRIPTION
**Pull request by allisonking**
Closes https://github.com/ethyca/fides/issues/3444

### Code Changes

* [x] Add breakpoints for banner width and borders at different sizes
* [x] Slightly edit the banner DOM to have a "container" element so that it's easy to center the banner when it isn't full width

### Steps to Confirm

* Spin up the privacy center and visit `localhost:3000/fides-js-demo.html` and resize your browser

### Pre-Merge Checklist

* [x] All CI Pipelines Succeeded
* Documentation:
  * [ ] documentation complete, [PR opened in fidesdocs](https://github.com/ethyca/fidesdocs/pulls)
  * [ ] documentation [issue created in fidesdocs](https://github.com/ethyca/fidesdocs/issues/new/choose)
* [ ] Issue Requirements are Met
* [ ] Relevant Follow-Up Issues Created
* [ ] Update `CHANGELOG.md`
* [ ] For API changes, the [Postman collection](https://github.com/ethyca/fides/blob/main/docs/fides/docs/development/postman/Fides.postman_collection.json) has been updated

### Description Of Changes

https://github.com/ethyca/fides/assets/24641006/f25e157a-f819-478f-b192-af28fc818c55

### CODE REVIEW

File: clients/privacy-center/public/fides-js-demo.html [hunk 1]
```diff
@@ -2,6 +2,7 @@
 <html>
   <head>
     <title>fides-js script demo page</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
```
Comment (eastandwestwind) [inline clients/privacy-center/public/fides-js-demo.html:5]: for consistency could we add this to `fides-js-components-demo.html` too?

**Review by eastandwestwind — CHANGES_REQUESTED** [review]

**Comment by NevilleS** [comment]
Oh, nice! The designs for this actually show that, when not full-width, there should be some spacing between the bottom of the page and the banner. Mockups vary between 51px and 66px which is confusing but we could pick `48px` and use that?

**Comment by allisonking** [comment]
> Oh, nice! The designs for this actually show that, when not full-width, there should be some spacing between the bottom of the page and the banner. Mockups vary between 51px and 66px which is confusing but we could pick `48px` and use that?

whoops good catch, didn't notice that!

**Comment by allisonking** [comment]
Added 48px spacing between bottom and the banner:

https://github.com/ethyca/fides/assets/24641006/394de60d-d7fe-4232-b86a-bd453e942c2a

**Review by eastandwestwind — APPROVED** [review]

**Pull request merged**